tiup: add tiup-command-env.md and tiup-command-help.md (#4960)
pingcap/docs: tiup/tiup-command-help.md

---
title: tiup help
---

# tiup help

> The TiUP command-line interface provides users with a wealth of help information. You can view them via the `help` command or the `--help` parameter.

Reviewer: ```suggestion
The TiUP command-line interface provides users with a wealth of help information. You can view it via the `help` command or the `--help` argument.
```

## Syntax

```sh
tiup help [command]
```

> `[command]` is used to specify the help information of which command that users need to view. If not specified, the help information of TiUP is viewed.

Reviewer: ```suggestion
`[command]` is used to specify the help information of which command that users need to view. If it is not specified, the help information of TiUP is viewed.
```
